mmpose/models/losses/classification_loss.py

```diff
@@ -190,9 +204,15 @@ def forward(self, pred_simcc, gt_simcc, target_weight):
             pred = pred.reshape(-1, pred.size(-1))
             target = target.reshape(-1, target.size(-1))
 
-            loss += self.criterion(pred, target).mul(weight).sum()
+            t_loss = self.criterion(pred, target).mul(weight)
+
+            if self.mask is not None:
+                t_loss = t_loss.reshape(N, K)
+                t_loss[:, self.mask] = t_loss[:, self.mask] * self.mask
```

Comment:
Should the last `self.mask` be replaced with `self.mask_weight`?

Reply:
You're right. I'll fix it soon.